graph_utility/time_memory_lines.py

```diff
@@ -22,9 +22,22 @@ def plot(data_list, test_names, graph_dir, metric):
     # Dataframe to hold data from all csvs
     combined_df = pd.DataFrame()
     for index, data in enumerate(data_list):
+        cutoff = -1
+        if metric == 'state space size':
+            cutoff = 0
+        if metric == 'verification time':
+            cutoff = 10
+        if metric == 'verification memory':
+            cutoff = 10000
```

Comment:
Do cutoff based on worst tests (maybe worst 2000) instead of fixed value

Reply:
is done